lib/protocol/crypto/src/binding.cc

```diff
@@ -84,11 +84,15 @@ class ChaChaPolyCipher : public ObjectWrap {
     SetPrototypeMethod(tpl, "encrypt", Encrypt);
     SetPrototypeMethod(tpl, "free", Free);
 
-    constructor().Reset(Nan::GetFunction(tpl).ToLocalChecked());
+    Local<Function> func = Nan::GetFunction(tpl).ToLocalChecked();
+    Local<Context> context = Nan::GetCurrentContext();
+    v8::Isolate* isolate = context->GetIsolate();
+
+    constructor().Set(isolate, func);
```

Comment:
I don't understand this change. `Reset()` was already [doing all of this](https://github.com/nodejs/nan/blob/8d312c2ae64240e76d8f4d428f73854455305567/nan_persistent_12_inl.h#L29).